src/main/python/deep_qa/training/trainer.py

```diff
@@ -236,17 +237,35 @@ def train(self):
 
         # First we need to prepare the data that we'll use for training.
         logger.info("Getting training data")
-        self.training_dataset = self._load_dataset_from_files(self.train_files)
+        if isinstance(self.train_files[0], list):
+            # We want to tune on other datasets.
+            self.training_datasets = []
```

Comment:
In order to avoid keeping lots of datasets around in memory, I would probably recommend loading the data in the training loop.  This interacts poorly with pre-training, though...

We need to think about how to fit the data indexer in cases like this.  Maybe leave a TODO about that.